src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -6082,5 +6082,35 @@ static void Main(string[] args)
 }";
             AssertFormat(code, code);
         }
+
+        [WorkItem(1298, "https://github.com/dotnet/roslyn/issues/1298")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void DontforceAccessorsToNewLineWithPropertyInitializers()
+        {
+            var code = @"using System.Collections.Generic;
+
+class Program
+{
+    public List<ExcludeValidationDescriptor> ValidationExcludeFilters { get; }
+    = new List<ExcludeValidationDescriptor>();
+}
+
+public class ExcludeValidationDescriptor
+{
+}";
+
+            var expected = @"using System.Collections.Generic;
+
+class Program
+{
+    public List<ExcludeValidationDescriptor> ValidationExcludeFilters { get; }
+    = new List<ExcludeValidationDescriptor>();
```

Comment:
What happens with this line? Does it preserve relative indentation?  This shows it at a relative indentation of zero, which I don't think is what people would want.